TodoList/Sources/TodoList.Repositories/RepositoryBootstrapper.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Microsoft.Practices.Unity;
+using TodoList.Contracts.Interfaces;
+
+namespace TodoList.Repositories
+{
+    public class RepositoryBootstrapper : IBootstrapper
+    {
+        public IUnityContainer Register(IUnityContainer container) 
+            => container.RegisterType<IListItemRepository, ListItemRepository>(new TransientLifetimeManager());
```

Comment:
This is not the correct lifetime manager. You need to use the one that is scoped (based on our verbal agreement and reasoning). Read more on [Unity lifetime manager](https://msdn.microsoft.com/en-us/library/ff660872(v=pandp.20).aspx), please.